### PR TITLE
Add tube back to qa.planx-pla.net

### PR DESCRIPTION
**Pull request by jawadqur**
### Environments
qa.planx-pla.net

### Description of changes
We need the tube image for the cloud-automation tests to complete. They rely on this manifest.

### CODE REVIEW

**Comment by PlanXCyborg** [comment]
# qa.planx-pla.net/manifest.json
## :warning: Services on branch
- jenkins
- jenkins-worker
- jenkins-ci-worker

File: qa.planx-pla.net/manifest.json [hunk 1]
```diff
@@ -23,7 +23,8 @@
     "revproxy": "quay.io/cdis/nginx:2022.03",
     "sheepdog": "quay.io/cdis/sheepdog:2022.03",
     "dashboard": "quay.io/cdis/gen3-statics:2022.03",
-    "metadata": "quay.io/cdis/metadata-service:2022.03"
+    "metadata": "quay.io/cdis/metadata-service:2022.03",
+    "tube": "quay.io/cdis/tube:2202.03"
```
Comment (atharvar28) [inline qa.planx-pla.net/manifest.json:27]: 2022.03

Comment (atharvar28) [inline qa.planx-pla.net/manifest.json:27]: 2022.03

**Comment by PlanXCyborg** [comment]
# qa.planx-pla.net/manifest.json
## :warning: Services on branch
- jenkins
- jenkins-worker
- jenkins-ci-worker

**Pull request merged**